coalib/processes/Processing.py

```diff
@@ -751,7 +755,12 @@ def execute_section(section,
                              results (bear names are key) as well as the
                              file dictionary.
     """
-    if debug:
+    debug_bears = (False
+                   if 'debug_bears' not in section or (
+                       section['debug_bears'].value == 'False') else (
```

Comment:
`section` offers a defaulted `get` function, so you can shorten code here I believe.
```python
typed_list(str)(section(...))
if bool(section.get('debug_bears', False)) else
False

Reply:
`bool(section.get('debug_bears', False))` will throw ValueError for the case `--debug-bears ABear` and If I do something like `bool(section.get('debug_bears', False).value)` then for the case `--debug-bears False` it will set the `debug_bears` value to `['False']` then in above code  I will have to check this case also. :sweat_smile: